Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/type/TaskType.java

```diff
@@ -59,12 +62,30 @@
         jsonObject.put(TaskTypeField.PROCESS_ID.getKey(), getId(task.getProcess()));
         jsonObject.put(TaskTypeField.PROCESS_TITLE.getKey(), getTitle(task.getProcess()));
         jsonObject.put(TaskTypeField.CLIENT_ID.getKey(), getClientId(task));
+        jsonObject.put(TaskTypeField.PROJECT_ID.getKey(), getProjects(task));
         jsonObject.put(TaskTypeField.TEMPLATE_ID.getKey(), getId(task.getTemplate()));
         jsonObject.put(TaskTypeField.TEMPLATE_TITLE.getKey(), getTitle(task.getTemplate()));
         jsonObject.put(TaskTypeField.ROLES.getKey(), addObjectRelation(task.getRoles()));
         return jsonObject;
     }
 
+    private List<Integer> getProjects(Task task) {
```

Comment:
Method is called getProjects but returns projects' ids.

Reply:
fixed